velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,97 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Support for Generic writer.
+
+
+Memory Pool
+=============
+
+* Memory code refactor.
+* Fix memory pool to always return valid buffer.
+* Fix memory usage count leak when async data cache retry transient allocation failure.
+* Make memory usage tracker thread safe.
+
+
+Presto Functions
+================
+
+* Add :func:`array_frequency`, :func:`json_parse`, :func:`shuffle`, :func:`sha1`
+  :func:`spooky_hash_v2_32`, :func:`spooky_hash_v2_64`, :func:`repeat`, :func:`week`,
+  :func:`week_of_year`, :func:`json_format` Presto functions.
+
+Spark Functions
+===============
+
+* Add support for DATE inputs to Spark's least and greatest functions.
+* Support spark-compatible year functions
+
+Hive Connector / S3 FileSystem
+==============================
+
+* Enable installSigPipeHandler option in S3FileSystem.
+* Move generic ScanSpec generation logic from HiveConnector to ScanSpec.
+* Support Map type in Parquet Reader.
+
+
+Substrait
+=========
+
+* Update substrait to 0.22.0.
+
+
+Performance and Correctness
+===========================
+
+* Skip evaluation for rows with errors in Expr::evalSimplified.
+* Fix unnecessary copy in the native Parquet reader.
+* Fix `createNullConstant`, `makeConstant`  for custom types.
+* Fix Common subexpression evaluation for Try.
```

Comment:
I suggest to replace these 4 items with one: "Fix multiple bugs in expression evaluation with TRY."